docs/navigating.md

```diff
@@ -127,6 +126,17 @@ class DetailsScreen extends React.Component {
 
 Another common requirement is to be able to go back _multiple_ screens -- for example, if you are several screens deep in a stack and want to dismiss all of them to go back to the first screen. In this case, we know that we want to go back to `Home` so we can use `navigate('Home')` (not `push`! try that out and see the difference). Another alternative would be `navigation.popToTop()`, which goes back to the first screen in the stack.
 
+Navigation actions behave idempotently when called with a key. When pushing, the navigator will push a route with that key. If the route already exists, it will navigate to that route and set the params.
```

Comment:
they also behave idempotently when not provided with a key in 2.0

Reply:
Updated